ci/test/pipeline.template.yml

```diff
@@ -582,6 +582,17 @@ steps:
       - ./ci/plugins/mzcompose:
           composition: storage-usage
 
+  - id: launchdarkly
+    label: "LaunchDarkly"
+    artifact_paths: junit_mzcompose_*.xml
+    depends_on: build-x86_64
+    timeout_in_minutes: 30
+    agents:
+      queue: linux-x86_64
+    plugins:
+      - ./ci/plugins/mzcompose:
+          composition: launchdarkly
```

Comment:
I think we might want to make this be a nightly and release qualification test instead? Otherwise I worry we'll be hitting the LD API quite a lot, and any flakiness in the LD API will block merging.

Reply:
Yes please make it Nightly and then I will take care of any flakes going forward.